Chat-Command-Reference.md

```diff
@@ -79,7 +79,7 @@ For example, the first 2 examples below are functionally equivalent.
 ```
 
 ## `//saplingaliases [aliases|all_saplings]`
-Lists all the currently registered sapling aliases in alphabetical order. These aliases can be used in the 
+Lists all the currently registered sapling aliases in alphabetical order. These aliases can be used in the
```

Comment:
Looks like maybe this line was never finished :P

Reply:
Oops! I must have been distrated by something in the middle of writing that. I've made a commit to fix that.